NDVI_seminat.Rmd

```diff
@@ -0,0 +1,989 @@
+  geom_point() +
+  facet_grid(tilstand ~ hovedtype)
+```
+2022 stands out with the highest NDVI values missing.
+WE omit 2022 for this analysis
+```{r}
+SentinelNDVI.seminat <- SentinelNDVI.seminat %>% filter(year != "2022")
+```
+
+# How do NDVI values vary between major and basic ecosystem types for sites in good condition? (using only NDVI data matching NiN-mapping years)
+```{r warning=FALSE}
+SentinelNDVI.seminat %>%
+  filter(tilstand == "God") %>%
+  filter(year == kartleggingsaar) %>%
+  ggplot(aes(x = hovedtype, y = mean)) +
+  geom_violin() +
+  geom_point(size = 0.7, shape = 16, color = "grey")
+
+SentinelNDVI.seminat$subtype <- factor(SentinelNDVI.seminat$subtype, levels = c("C1", "C2", "C3", "C4", "C5", "C6", "C7", "C8", "C9", "C10", "C11", "C12", "C13", "C14", "C15", "C16", "C17", "C18", "C19", "C20", "C21", "E1", "E2", "E3", "E4", "E5", "E6", "E7"))
+SentinelNDVI.seminat %>%
+  filter(tilstand == "God") %>%
+  filter(year == kartleggingsaar) %>%
+  ggplot(aes(x = subtype, y = mean)) +
+  geom_violin() +
+  geom_point(size = 0.7, shape = 16, color = "grey") +
+  facet_wrap(~hovedtype) +
+  theme(axis.text.x = element_text(angle = -45, vjust = 0.5, hjust = 0.2, size = 6))
+```
+NDVI varies between major ecosystem types with the highest values in semi-natural grasslands (T32) and in previously intensively used and altered grasslands that today resemble  a semi-natural grassland (T40, T41), lowest values in coastal heathlands (T34). Other ecossytem types: Boreal heath (T31), semi-natural coastal grasslands (T33). The overlaps between the ecosystem types are considerable.
+With respect to basic ecosystem types within the major types, NDVI appears to cover very similar ranges across the subtypes, and those with deviating violin plots have a smaller sample size. There is no obvious trend related to the structuring environmental gradients withing the major types, and also here the overlap is large. 
+
+# How does NDVI vary across regions in semi-naturals (major types) in good condition? (using only NDVI data matching NiN-mapping years)
+```{r warning=FALSE}
+SentinelNDVI.seminat %>%
+  filter(tilstand == "God") %>%
+  filter(year == kartleggingsaar) %>%
+  filter(!is.na(region)) %>%
+  ggplot(aes(x = region, y = mean)) +
+  geom_violin() +
+  geom_point(size = 0.7, shape = 16, color = "grey") +
+  facet_wrap(~hovedtype) +
+  theme(axis.text.x = element_text(angle = -45, vjust = 0.5, hjust = 0.2))
+```
+For the meadow systems (T32, T33, T40, T41) there is no obvious trends in geography. In the heath systems (T31, T34) the distribution of NDVI values in Central Norway is somewhat lower than in the other regions.
+
+# How does NDVI vary across condition classes (for major types and NDVI data matching NiN-mapping years)
+```{r warning=FALSE}
+SentinelNDVI.seminat %>%
+  filter(year == kartleggingsaar) %>%
+  ggplot(aes(x = tilstand, y = mean)) +
+  geom_violin() +
+  geom_point(size = 0.7, shape = 16, color = "grey") +
```

Comment:
ok

Reply:
in previous plots I used 

theme(axis.text.x = element_text(angle = -45, vjust = 0.5, hjust = 0.2))

I'll make sure to use the same throughout the document, otherwise it will look odd :)

Reply:
Sounds good. I made some temporary edits that you might have to adjust then.